### ISSUE TITLE
[DataTable] OverflowMenu doesn't have keyboard accessible styles

### ISSUE THREAD

**Issue by diego-codes** [carbon-design-system/carbon#3372]
## Detailed description

When using an `OverflowMenu` component inside of a `DataTable` as highlighted [here](https://www.carbondesignsystem.com/components/data-table/usage#inline-actions), the keyboard actions of the `OverflowMenu` do not work. 

Once the menu is open, the user is not able to use their arrow keys to select an `OverflowMenuItem` option as one would be able to in an `OverflowMenu` that is not inside of a `DataTable` because there are no visual indicators of which option is currently active.

It is expected that a user would be able to make a selection inside of an `OveflowMenu` in a `DataTable` just using their keyboard.

> What version of the Carbon Design System are you using?
Version 10

## Steps to reproduce the issue

1. Go to sandbox https://codesandbox.io/s/carbon-react-datatable-overflowmenu-a11y-bug-c92rz
2. Using the keyboard only, open any overflow menu on the table
3. Use arrow keys to attempt to select an action
4. Notice that there are no visual indicators of which action is active to be selected

## Additional information

**Keyboard usage of standalone `OverflowMenu`**
![Kapture 2019-07-11 at 15 36 36](https://user-images.githubusercontent.com/5973294/61083551-b162d180-a3f1-11e9-9963-2bce349fc077.gif)

**Keyboard usage of `OverflowMenu` inside of a `DataTable`**
![Kapture 2019-07-11 at 15 38 15](https://user-images.githubusercontent.com/5973294/61083652-ee2ec880-a3f1-11e9-890d-ce0ef151f68d.gif)

**Comment by asudoh** [comment]
Hi @diego-codes does your overflow menu in data table have `data-floating-menu-primary-focus` attribute?

**Comment by diego-codes** [comment]
Hi @asudoh, no. I searched the entire DOM for this attribute and nothing came up.

**Comment by asudoh** [comment]
OK would you want to try adding that attribute to the focusable DOM node in the first menu item?

**Comment by diego-codes** [comment]
Adding the `data-floating-menu-primary-focus` attribute to the DOM node works! It appears that it is not applied to the menu in the `DataTable` by default then.

**Comment by jendowns** [comment]
Related to #2481
